cmd/workerpodautoscaler/run.go

```diff
@@ -59,7 +60,7 @@ func (v *runCmd) new() *cobra.Command {
 	flags.String("kube-config", "", "path of the kube config file, if not specified in cluster config is used")
 	flags.Int("sqs-short-poll-interval", 20, "the duration (in seconds) after which the next sqs api call is made to fetch the queue length")
 	flags.Int("sqs-long-poll-interval", 20, "the duration (in seconds) for which the sqs receive message call waits for a message to arrive")
-
+	flags.Int("beanstalk-poll-interval", 3, "the duration (in seconds) for which the beanstalk receive message call waits for a message to arrive")
```

Comment:
need to have a flag to choose which all queue provider the WPA should have, a list like 
`queue-providers`.  And we should create and start the pollers for only those queue providers. Because some people would just want to run SQS and some just beanstalk and some might want both.

Reply:
It sounds good. Sure will implement this.

Reply:
Implemented this